data-es2016plus.js

```
@@ -6472,7 +6472,7 @@ exports.tests = [
         */},
         res: {
           safari15: true,
-          chrom100: true,
```

We should probably throw on unknown browsers in `res`.

---

sounds like a great improvement